dev_guide/projects.adoc

```diff
@@ -165,3 +165,36 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
```

Comment:
@sosiouxme Yes, i brought it up with the team a few updates ago, but got no response. I can worry about that when we get to peer review. As long as all the info is there for now.